lib/tasks/glueby/contract/wallet_adapter.rake

```diff
@@ -0,0 +1,42 @@
+module Glueby
+  module Contract
+    module Task
+      module WalletAdapter
+        def import_block(block_hash)
+          block = Glueby::Internal::RPC.client.getblock(block_hash, 0)
+          block = Tapyrus::Block.parse_from_payload(block.htb)
+          block.transactions.each { |tx| import_tx(tx.txid) }
```

Comment:
Do you need to set (Database) transaction block to roll back in the event of an unexpected error?

Reply:
Yes. Fixed at 93a657d.